### ISSUE TITLE
Add Event Tracking for the Locations Page

### ISSUE THREAD

**Issue by martypowell** [baltimorecounty/BCPL-assets#500]
Events to track

- Map Marker Click
  - action: "Map Marker Click"
  - category: "BCPL Locations"
  - label: the location name
- Filter List Checkbox Click 
  We want these the same as events so use this
  ```javascript
  trackEvent({
   action: `Filter Selection`,
    category: CONSTANTS.analytics.bcplLocationsCategory,
    label: `${name} - ${isChecked ? 'Selected' : 'Unselected'}`
  });
  ```
  Note: bcplLocationsCategory needs to be added to the constants and should be "BCPL Locations"

- Filter Tag List
  - action: "Filter Tag Selection"
  - category: "BCPL Locations"
  - label: the location name + the tag name + selected/unselected

- Clear All Filters (see [BCPL Events Tracking](https://github.com/baltimorecounty/BCPL-assets/issues/495))
- Contact Email Button Selection
- Contact Phone Button Selection

**Comment by martypowell** [comment]
@danfox01 as we discussed, the bug jason found is related to debugging on the developers end. We are going to leave for now.

**Comment by danfox01** [comment]
Resolved with 1.5.4
